packages/viewer/src/view-2d.ts

```diff
@@ -89,7 +90,13 @@ export const view2d = setup({
       const bounds = await image.getWorldBounds(image.coarsestScale);
       const { pose: currentPose, verticalFieldOfView } =
         camera.getSnapshot().context;
-      const pose = reset2d(currentPose, verticalFieldOfView, bounds, aspect);
+      // rotate so to look down -Z with +Y down on screen
```

Comment:
+Y should be up -- I am not sure if this comment and the PR title just need to be updated.

The change should probably be limited to vtk.js 2d viewport. Ref:

https://www.slicer.org/wiki/Slicer:VTK_ITK_Coordinates

Reply:
I was incorrect about the world +Z axis.  This rotation points world +Z into the screen.  Comment changed to:
`// rotate so +Z goes into screen and +Y is down on screen`